main.go

```diff
@@ -257,6 +260,17 @@ func main() {
 	cmd.SetDir(workDir)
 
 	if err := cmd.Run(); err != nil {
+		fmt.Println()
+		log.Errorf("Fastlane command: (%s) failed", command.PrintableCommandArgs(true, fastlaneCmd))
+		log.Errorf("See the error log below, and use it to send issue report to fastlane github issue tracker:")
+		log.Printf("https://github.com/fastlane/fastlane/blob/master/.github/ISSUE_TEMPLATE.md#environment")
+		fmt.Println()
+
+		fastlaneEnvLogCmd := []string{"fastlane", "env"}
+		inputReader := strings.NewReader("n")
+		if err := command.RunCommandWithReaderAndWriters(inputReader, os.Stdout, os.Stdout, fastlaneEnvLogCmd[0], fastlaneEnvLogCmd...); err != nil {
```

Comment:
Use `rubycommand.New()` to generate the command model.
Do not forget to:

- set the work dir
- set stdin (`n`)
- and `cmd.SetStdout(os.Stdout).SetStderr(os.Stderr)`